catalog/ACA/ACA_reef_habitat_v2_0.jsonnet

```diff
@@ -0,0 +1,320 @@
+local id = 'ACA/reef_habitat/v2_0';
```

Comment:
could you add successor / predecessor / latest link to both old and new ACA entries similarly to the two files in https://github.com/google/earthengine-catalog/tree/main/catalog/CAS ?